packages/web/public/static/locales/de/notifications.json

```diff
@@ -0,0 +1,21 @@
+{
+  "feedTitle": "Notifications",
+  "emptyFeed": "You have no notifications... yet!",
+  "levelOne": {
+    "newFollowers": "You got {{followerCount}} new follower",
+    "threadComments": "You got {{commentCount}} new feedback comments on this post",
+    "postComments": "You got {{commentCount}} new general comments on this post",
+    "postClaps": "{{clapCount}} people clapped for your post",
+    "threadCommentThanks": "{{userIdentifier}} said thanks for {{thanksCount}} feedback comments",
+    "newPosts": "Journalers you follow published {{postCount}} new posts"
+  },
+  "levelTwo": {
+    "newFollowers": "These {{followerCount}} awesome Journalers are now following you",
+    "threadCommentThanks": "You received {{thanksCount}} thanks for feedback on this post",
+    "postClaps": "These {{userCount}} people clapped for your post"
```

Comment:
```suggestion
    "postClaps": "Diese {{userCount}} Leute haben deinem Beitrag applaudiert"
```

Reply:
Plural issue: Leute and people can't be a single person. 

1. This person clapped for your post = Diese Person hat deinem Beitrag applaudiert
2. These 3 people clapped for your post = Diese 3 Leute haben deinem Beitrag applaudiert